recipe/patches/0001-Force-usage-of-python-3-and-error-without-numpy.patch

```diff
@@ -32,3 +32,6 @@ index e78305e0a8e..15c62548601 100644
          caffe2_update_option(USE_NUMPY OFF)
        else()
          caffe2_update_option(USE_NUMPY ON)
+-- 
+2.48.1
+
```

Comment:
If you do generate patches, please use `git format-patch <hash> --no-signature` (also for future reference; no change necessary here)

Reply:
Okay, thanks.

```
git config --global format.signature ''
```

should resolve it going forward.